.gitignore

```diff
@@ -1,7 +1,3 @@
-bin/
 build/
-lib/
-lib64
-pyvenv.cfg
-share/
 sw/build/
+venv/
```

Comment:
Maybe we would prefer making this a hidden directory (`.venv`), which [is common](https://docs.python.org/3/tutorial/venv.html#creating-virtual-environments)?

Reply:
Good suggestion! done